DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/component_interfaces_comparison/mm-host/src/vector_add.cpp

```diff
@@ -0,0 +1,141 @@
+#include <iostream>
+
+// oneAPI headers
+#include <sycl/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include "exception_handler.hpp"
+
+// Buffer locations for MM Host interfaces
+constexpr int kBL0 = 0;
```

Comment:
In rel/2024.0 we decided to reserve BL 0 for raw pointers. This is due to a bug we have with the raw pointer. So we need to use BL starting from 1